pkgs/top-level/all-packages.nix

```diff
@@ -30637,6 +30637,8 @@ with pkgs;
   prevo-data = callPackage ../applications/misc/prevo/data.nix { };
   prevo-tools = callPackage ../applications/misc/prevo/tools.nix { };
 
+  protonup-qt = callPackage ../applications/misc/protonup-qt { };
```

Comment:
Do appimage packages get the -bin suffix?

Reply:
Good question, that would make sense. I'll do some research.

Reply:
It can't seem to find a formal ruling on it, and package names of other appimage wrappers seem to be 50/50, but I'm going to change it because it makes more sense to me.